.github/workflows/ci.yml

```diff
@@ -119,8 +121,8 @@ jobs:
     - uses: lukka/get-cmake@latest
     - uses: lukka/run-vcpkg@v11
       with:
-        vcpkgGitCommitId: 'c9f906558f9bb12ee9811d6edc98ec9255c6cda5'
-        prependedCacheKey: vs-x64
+        vcpkgGitCommitId: '69efe9cc2df0015f0bb2d37d55acde4a75c9a25b'
+        doNotCache: true
```

Comment:
why not cache in the CI workflow, as opposed to CD?

Reply:
Both CI and CD shouldn't cache in this case. It disables caching of a github repo + a 6MB executable. In total it's 70MB in caches for not much benefit, because recloning and redownloading is just as fast as restoring this 70MB cache. There's a discussion upstream about whether to remove this caching altogether, so for now let's keep it disabled to save cache space.